libs/datamodel/core/tests/validation/composite_types/ignore_field_attribute_is_not_allowed.prisma

```diff
@@ -15,9 +15,10 @@ model B {
 
 
 
+
 // [1;91merror[0m: [1mAttribute not known: "@ignore".[0m
 //   [1;94m-->[0m  [4mschema.prisma:8[0m
 // [1;94m   | [0m
 // [1;94m 7 | [0m  name String
-// [1;94m 8 | [0m  c String @[1;91mignore[0m
+// [1;94m 8 | [0m  c String [1;91m@ignore[0m
```

Comment:
Unrelated change?

Reply:
This one is a bit of a mystery to me actually, I need to have a look at what triggered it.

Reply:
https://github.com/prisma/prisma-engines/pull/3123#discussion_r945864999